src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -158,6 +159,20 @@ public class DurableTaskExtension :
         /// </value>
         public bool DisableHttpManagementApis { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value of Event Grid Topic Endpoint for emitting lifecycle events.
+        /// If this property has been set, it will emit the event to the Event Grid Topic.

```

Comment:
I word reword this slightly:
*Gets or sets the URL of an Azure Event Grid custom topic endpoint. When set, orchestration life cycle notification events will be automatically published to this endpoint.*

Reply:
Thank you! We will modify it.